src/services/multiplayerGameService.js

```diff
@@ -0,0 +1,102 @@
+import config from '../../config';
```

Comment:
If you use the same fetch method with the same header... you should create a specific http client for that instead of create it each time